client/src/components/Header.css

```diff
@@ -0,0 +1,9 @@
+.header {
+    background-color: #AFD0FF;
+    padding: 10px 20px;
+    color: rgb(0, 0, 0);
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
```

Comment:
For more formatting, you could add the title like `.title` so in your js file you can have similar formatting like the figma design.